pytest.ini

```diff
@@ -1,3 +1,7 @@
 [pytest]
 DJANGO_SETTINGS_MODULE=osf_tests.settings
 addopts = --tb=short --reuse-db
+filterwarnings = 
+    ignore::UserWarning
```

Comment:
Again, I'm concerned that we might miss out on legitimate warnings if we do this; maybe `once` would be better?